src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -585,6 +629,14 @@ private static void checkScreenCaptureAllowed() {
         }
     }
 
+    /**
+     * Resets the stored screen data capture permission for a set of screens.
```

Comment:
Does "reset" here mean that the next attempt to capture screen pixels will require another permission? It's not clear what the effects of this "reset" are.

Also, the part "for a set of screens" kind of implies that for some other set of screens the permission will not be reset, but how can one specify for which set of screens the permission is to be reset? Perhaps, simply
"Revokes the stored permission to capture screen data. Subsequent calls to getPixelColor and createScreenCapture may request a new permission from the user on applicable platforms"?

Reply:
Yes, this sounds much better, updated.